scripts/fhir-works-deploy.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -e
+
+read -n 1 -p "This script adds local dependencies to your 'fhir-works-on-aws-deployment' package during build. It will then revert your 'package.json' to the last version of the package in Git. Have you committed the 'package.json' file in your 'fhir-works-on-aws-deployment' package  to Git (y/n)?" answer
+if [ $answer != "y" ]
+then
+  printf "\nPlease commit the package.json file in  your 'fhir-works-on-aws-deployment' package to Git"
+  exit
+fi
+
+packages=(
+"fhir-works-on-aws-authz-rbac"
+"fhir-works-on-aws-persistence-ddb"
+"fhir-works-on-aws-routing"
+"fhir-works-on-aws-search-es"
+)
+
+# Go into interface package, delete old pregenerated files, install all dependencies again, and then push the package to yalc
+cd fhir-works-on-aws-interface
+rm -rf node_modules build/* dist/* 
+yarn install
+yalc push
+
+cd ..
+
+printf "\nFor each package except for deployment and interface package, add interface package as a dependency, remove old pregenerated files, install all dependencies again, and push new packages to yalc\n"
+for i in "${packages[@]}"
+do
+    cd $i
+    rm -rf node_modules build/* dist/* 
```

Comment:
Is it necessary to delete node_modules and the re-run `yarn install`? This makes the whole process much slower. 

I know that yalc + serverless gets messy and starting on a clean slate makes things easier. But It'd be great to have this script run faster.

Reply:
Without deleting `node_modules` and re running `yarn install`, the code would sometimes not pick up new changes to the dependent packages. Unfortunately, although this is slower, it's the most reliable way that I've found to make the deploy process work.